form/validation_groups.rst

```diff
@@ -32,7 +32,9 @@ In both of these cases, *only* the ``registration`` validation group will
 be used to validate the underlying object. To apply the ``registration``
 group *and* all constraints that are not in a group, use::
 
-    'validation_groups' => ['Default', 'registration']
+    [
+        'validation_groups' => ['Default', 'registration']
+    ];
```

Comment:
I'm not really sure I like this, I think it's a bit harder to understand than the previous code. More over, the code example is highlighted correctly by Pygments: https://symfony.com/doc/current/form/validation_groups This is often not the case with syntax errors, so if it is I'm OK if we break syntax to make code examples easier to understand :)

Reply:
I think your argumentation make sense. It made me realise that we cannot be 100% correct syntax. We need our syntax checker to work with a "baseline". 

I'll revert.